apps/frontend/src/components/Inbox/ScheduleItem.tsx

```
@@ -0,0 +1,202 @@
+    if (matchingMonths.length === 0) {
+      setDueDates(generateDefaultDates()) // Fallback to default dates
+      return
+    }
+
+    const generatedDates: DueDate[] = []
+    const now = new Date()
+
+    if (dayPart) {
+      const day = parseInt(dayPart, 10)
+
+      if (isNaN(day) || day < 1 || day > 31) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)